### PR TITLE
Upgrade to Liquibase 3.6.2

### PR DESCRIPTION
**Pull request by 72MiguelGomes**
The new version of SpringLiquibase was not released yet, but will add this PR here, and as soon as the new version is available we just need to update it and merge.

Issue: https://github.com/spring-projects/spring-boot/issues/13550.

This PR should keep on hold for a little. @isopov as soon as the new version was released please update us.

### CODE REVIEW

File: spring-boot-project/spring-boot-dependencies/pom.xml [hunk 1]
```diff
@@ -118,7 +118,7 @@
 		<kafka.version>1.1.0</kafka.version>
 		<kotlin.version>1.2.41</kotlin.version>
 		<lettuce.version>5.1.0.M1</lettuce.version>
-		<liquibase.version>3.6.1</liquibase.version>
+		<liquibase.version>3.6.2-SNAPSHOT</liquibase.version>
```
Comment (72MiguelGomes) [inline spring-boot-project/spring-boot-dependencies/pom.xml]: Please notice this was used just to test, we must remove the `SNAPSHOT`as soon as the new versions is available

**Comment by isopov** [comment]
@72MiguelGomes 3.6.2 liqubiase version has been released

File: spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/liquibase/LiquibaseProperties.java [hunk 2]
```diff
@@ -53,6 +53,26 @@
...
+	 */
+	private String liquibaseSchema;
+
+	/**
+	 * Liquibase tablespace.
+	 */
+	private String liquibaseTablespace;
+
+	/**
+	 * Database changelog table.
```
Comment (snicoll) [inline spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/liquibase/LiquibaseProperties.java]: table name?

**Comment by 72MiguelGomes** [comment]
@snicoll thanks for the feedback, will update it asap.

File: spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/liquibase/LiquibaseProperties.java (hunk 2 above)
Comment (72MiguelGomes) [reply spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/liquibase/LiquibaseProperties.java]: What do you mean? I am using the same that used in liquibase project

**Comment by 72MiguelGomes** [comment]
@snicoll Those values does not have any default value define, so there is no problem.

https://github.com/liquibase/liquibase/blob/master/liquibase-core/src/main/java/liquibase/integration/spring/SpringLiquibase.java#L81

Regarding de adoc, do you know if is possible generate that automatically during the build or is it always a manual process?

@philwebb The PR is ready to merge, please let me know if you find something that is missing.

@isopov Thanks for letting me know about liquibase release.

**Comment by snicoll** [comment]
> @snicoll Those values does not have any default value define, so there is no problem.

uh? Surely there must be some sort of default somewhere right?

> Regarding de adoc, do you know if is possible generate that automatically during the build or is it always a manual process?

That's how it is right now, you can subscribe to #8237.

File: spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/liquibase/LiquibaseProperties.java (hunk 2 above)
Comment (snicoll) [reply spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/liquibase/LiquibaseProperties.java]: I mean that the description can be more informative.

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc [hunk 3]
```diff
@@ -566,6 +566,10 @@ content into your application. Rather, pick only the properties that you need.
 	spring.liquibase.check-change-log-location=true # Whether to check that the change log location exists.
 	spring.liquibase.contexts= # Comma-separated list of runtime contexts to use.
 	spring.liquibase.default-schema= # Default database schema.
+	spring.liquibase.liquibase-schema= # Liquibase database schema.
+  spring.liquibase.liquibase-tablespace= # Liquibase tablespace.
```
Comment (snicoll) [inline spring-boot-project/spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc]: There is an indentation problem here.

File: spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/liquibase/LiquibaseProperties.java (hunk 2 above)
Comment (snicoll) [reply spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/liquibase/LiquibaseProperties.java]: In `GlobalConfiguration` the description is 

> Name of table to use for tracking change history

which is way more informative than the Javadoc. I prefer to use that

**Comment by snicoll** [comment]
@72MiguelGomes please don't act on the default value as I'd like to do this globally (https://github.com/spring-projects/spring-boot/issues/13765). 

Can you please fix the indentation and use a better description for those new keys?

File: spring-boot-project/spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc (hunk 3 above)
Comment (72MiguelGomes) [reply spring-boot-project/spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc]: Thanks

**Comment by snicoll** [comment]
@72MiguelGomes thank you, this is now merged in `master`.

**Comment by DanielFran** [comment]
@snicoll Is it possible to include this to next 2.0.4 release too?

**Comment by wilkinsona** [comment]
Unfortunately not. 2.0.x uses Liquibase 3.5.x and we won't upgrade to a new minor release of Liquibase in a maintenance release of Spring Boot.

**Pull request closed without merging**